core/src/main/java/io/aiven/klaw/helpers/HandleDbRequests.java

```diff
@@ -119,9 +120,29 @@ List<AclRequests> getAllAclRequests(
       boolean showRequestsOfAllTeams,
       int tenantId);
 
+  List<AclRequests> getAllAclRequestsFiltered(
+      boolean allReqs,
+      String requestor,
+      String role,
+      String status,
+      boolean showRequestsOfAllTeams,
+      String topic,
+      String environment,
+      AclType aclType,
+      int tenantId);
+
   List<AclRequests> getCreatedAclRequestsByStatus(
       String requestor, String status, boolean showRequestsOfAllTeams, int tenantId);
 
+  List<AclRequests> getCreatedAclRequestsByStatus(
```

Comment:
there is old unused method getCreatedAclRequestsByStatus with 4 params, can be deleted ?